.changeset/new-starfishes-glow.md

```diff
@@ -0,0 +1,8 @@
+---
+'@vocab/types': patch
+'@vocab/core': patch
+---
+
+Remove [unused] `ICUFormatResult` type
+
+[unused]: https://github.com/search?q=+%2FICUFormatResult%2F&type=code
```

Comment:
Same on this, could it be moved to a separate PR that just removes it from `@vocab/types`?

Reply:
Nobody uses that type. https://github.com/search?q=+%2FICUFormatResult%2F&type=code

Reply:
Sure, but I still feel like it doesn't belong in this PR.